fbchat/_client.py

```diff
@@ -976,6 +976,42 @@ def getUserActiveStatus(self, user_id):
         """
         return self._buddylist.get(str(user_id))
 
+    def _fetchImages(self, thread_id):
+        data = {"id": thread_id, "first": 12}
+        j = self.graphql_request(_graphql.from_query_id("515216185516880", data))
+        to_continue = True
+        while to_continue:
+            page_info = j[thread_id]["message_shared_media"]["page_info"]
+            end_cursor = page_info.get("end_cursor")
+            try:
+                yield j[thread_id]["message_shared_media"]["edges"][0]
+                del j[thread_id]["message_shared_media"]["edges"][0]
+            except IndexError:
+                if page_info.get("has_next_page"):
+                    data["after"] = end_cursor
+                    j = self.graphql_request(
+                        _graphql.from_query_id("515216185516880", data)
+                    )
+                else:
+                    to_continue = False
+
+    def fetchThreadImages(self, thread_id=None):
+        """
+        Creates generator object for fetching images posted in thread.
```

Comment:
Could you add some example usage to `examples/fetch.py`, perhaps along with usage of [`itertools.islice`](https://docs.python.org/3/library/itertools.html#itertools.islice) to fetch e.g. the first 20 images.

Reply:
Yup. Easy!